src/fuzzer/fuzzers.h

```diff
@@ -33,16 +33,16 @@ extern "C" int LLVMFuzzerInitialize(int*, char***) {
 // Called by main() in libFuzzer or in main for AFL below
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t in[], size_t len) {
    if(len <= max_fuzzer_input_size) {
-      fuzz(in, len);
+      fuzz( std::span<const uint8_t> ( in, len ) );
    }
    return 0;
 }
 
 // Some helpers for the fuzzer jigs
 
-inline std::shared_ptr<Botan::RandomNumberGenerator> fuzzer_rng_as_shared() {
-   static std::shared_ptr<Botan::ChaCha_RNG> rng =
-      std::make_shared<Botan::ChaCha_RNG>(Botan::secure_vector<uint8_t>(32));
+inline std::shared_ptr<Botan::System_RNG> fuzzer_rng_as_shared() {
+   static std::shared_ptr<Botan::System_RNG> rng =
+      std::make_shared<Botan::System_RNG>();
```

Comment:
This definitely doesn't work - the non-determinism of the RNG will confuse the fuzzer. Lets revert this bit and then take up the RNG piece in a future PR. I'll add more context in the original ticket

Reply:
Okay, thank you. I changed it in my recent commit.